dist/profile/templates/jenkinscontroller/casc/clouds.yaml.erb

```
@@ -259,6 +259,13 @@ jenkins:
         <%- end -%>
           slaveConnectTimeout: 100
           yamlMergeStrategy: "override"
+          volumes:
+          - emptyDirVolume:
+              memory: false
+              mountPath: "/tmp"
+          - emptyDirVolume:
+              memory: false
+              mountPath: "/home/jenkins/.m2"
```

```suggestion
              mountPath: "/home/jenkins/.m2/repository"
```

To avoid having `settings.xml` with sensitive data to accidentally be persisted on the underlying host.